package.json

```diff
@@ -18,7 +18,7 @@
     "karma-spec-reporter": "0.0.13",
     "karma-osx-reporter": "*",
     "mocha": "~1.15.1",
-    "phantomjs": "1.9.7-1",
```

Comment:
good stuff. Should ensure that we never get caught again with phantomjs upgrades